pfcpiface/messages.go

```diff
@@ -321,21 +336,26 @@ func (pc *PFCPConn) handleSessionModificationRequest(upf *upf, msg message.Messa
 
 	addPDRs := make([]pdr, 0, MaxItems)
 	addFARs := make([]far, 0, MaxItems)
-
+	var fseidIP uint32
+	_ = binary.Read(bytes.NewBuffer(net.ParseIP(sourceIP).To4()),
+		binary.LittleEndian, &fseidIP)
 	for _, cPDR := range smreq.CreatePDR {
 		var p pdr
 		if err := p.parsePDR(cPDR, localSEID, pc.mgr.appPFDs); err != nil {
 			return sendError(err)
 		}
+		p.fseidIP = fseidIP
 		session.CreatePDR(p)
 		addPDRs = append(addPDRs, p)
 	}
 
 	for _, cFAR := range smreq.CreateFAR {
 		var f far
 		if err := f.parseFAR(cFAR, localSEID, upf, create); err != nil {
-			return sendError(err)
+			log.Println("FAR Parse failed ", err)
+			continue
```

Comment:
why are progressing when there is an error? shouldnt we report the error

Reply:
we are just printing the error here.. even if one of the far is proper we still go ahead and write to fast path. if all FAR parse fails then we respond with error.

Reply:
doesnt sound right. anything in the spec? 

/cc @thakurajayL

Reply:
control plane wrongly thinks all succeeded?

Reply:
discussed with Ajay and removed the skipping of failure part.